.github/workflows/main.yml

```diff
@@ -58,6 +58,10 @@ jobs:
       # Print resolved dotnet info
```

Comment:
@atmgrifter00 @msmithNI we should try to get these PRs closed quicker, the goal is within a day or two, I made the following two changes:

1. I pinned the playwright version, i.e. `<PackageReference Include="Microsoft.Playwright" Version="[1.42.0]" />` while not strictly necessary to hard code it I think the rennovate interactions are poor with the current approach that tries to generate the version from the package.json since that does not run when rennovate runs. I just removed that complexity and hard coded it. Don't think it'll be that hard to bump when the version is bumped elsewhere. It also was previously not actually formatted as a strict version and now is, i.e. `Version="[1.42.0]"` vs `Version="1.42.0"`.
2.  Looks like the NI.CSharp.Analyzers version was causing problems that were not resolved quickly I just pinned to the old version, i.e. `<PackageReference Include="NI.CSharp.Analyzers" Version="[2.0.21]" />` and filed https://github.com/ni/nimble/issues/2014 which can be addressed in parallel.

It's not a problem to spend a little time trying to resolve the issues but should try and avoid keeping them open for long periods.